### PR TITLE
fix typo and shorten line length in faq.md

### PR DESCRIPTION
**Pull request by dkarrasch**
introduced in #33871

### CODE REVIEW

**Comment by StefanKarpinski** [comment]
Hah. I did virtually the same fix: #34144.

File: doc/src/manual/faq.md [hunk 1]
```diff
@@ -627,8 +627,11 @@ have, it ends up having a substantial cost due to compilers (LLVM and GCC) not g
 around the added overflow checks. If this improves in the future, we could consider defaulting
 to checked integer arithmetic in Julia, but for now, we have to live with the possibility of overflow.
 
-In the meantime, overflow-safe integer operations can be achieved through the use of external libraries such as [SaferIntegers.jl](https://github.com/JeffreySarnoff/SaferIntegers.jl). Note that, as stated previously, the use of these libraries signically increases the execution time of code using the checked integer types. However, for limited usage, this is far less of an issue than if it were used for all integer operations. You can follow the status of the discussion [here](https://github.com/JuliaLang/julia/issues/855).
-
+In the meantime, overflow-safe integer operations can be achieved through the use of external libraries such
+as [SaferIntegers.jl](https://github.com/JeffreySarnoff/SaferIntegers.jl). Note that, as stated previously,
+the use of these libraries significally increases the execution time of code using the checked integer types.
```
Comment (rfourquet) [inline doc/src/manual/faq.md]: significally :)

Comment (StefanKarpinski) [reply doc/src/manual/faq.md]: Yo dawg, I heard you like typos in your typo fixes...

**Comment by dkarrasch** [comment]
> Hah. I did virtually the same fix: #34144.

... except for replacing one typo by another. 🤣 I'll close this, and try to sneak in another typo in a different PR.

**Pull request closed without merging**